pom.xml

```diff
@@ -54,6 +56,26 @@
     </repository>
   </repositories>
 
+  <dependencyManagement>
```

Comment:
This could all be handled in common - if you override the elastic-search-version property here it will update the equivalent imports in the common module, but I'm not sure why you would want different versions than specified in common?